Program.cs

```diff
@@ -55,6 +59,7 @@ public static void AddPerson()
             people.Add(newPerson);
```

Comment:
You don't need this SetId method. You can send new id with new name and new surname to the constructor together

Reply:
done 👍